solutions/cf-1208F.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: cf-1208F
+source: CF
+title: Bits and Pieces
+author: Dong Liu
+---
+
+## Explanation
+
+The answer is equivalent of enumerating $a_i$ and finding the optimal $x$ where $x=a_j\&a_k$ and $i<j<k$ and $a_i|x$ is maximum.
```

Comment:
what's the "optimal" x?

Reply:
maybe denote somewhere that | is the bitwise OR, thought it was divisibility for a second

Reply:
> what's the "optimal" x?

and $a_i|x$ is maximum.

Reply:
what does that mean?